dpe/src/commands/derive_child.rs

```diff
@@ -142,7 +142,10 @@ impl DeriveChildCmd {
         let default_context_idx = dpe
             .get_active_context_pos(&ContextHandle::default(), target_locality)
             .ok();
-        let num_contexts_in_locality = dpe.count_active_contexts_in_locality(target_locality)?;
+
+        let active_contexts_in_locality_pred =
+            |c: &Context| c.state == ContextState::Active && c.locality == target_locality;
+        let num_contexts_in_locality = dpe.count_contexts(active_contexts_in_locality_pred)?;
```

Comment:
nit (plus whatever formatting `cargo fmt` suggests)

```suggestion
        let num_contexts_in_locality = dpe.count_contexts(|c: &Context| c.state == ContextState::Active && c.locality == target_locality)?;
```

Reply:
Done